test/integration.js

```diff
@@ -32,6 +32,7 @@ describe("order", function () {
 
   describe("create order", function () {
     it("should return a order object", async () => {
+      await delay(1000); // to avoid rate limit
```

Comment:
this is not a good practice - making the tests run slow = bad develop experience. Can you avoid the rate limiter in your tests instead?

Reply:
+1

Reply:
@dannybabbev @bruffstar Removed the delay and refactored inscription test cases.